### PR TITLE
Catalan translation update

### PR DESCRIPTION
**Pull request by jguille2**
Just a Catalan translation update :)

Ping @bromagosa to check two things:
  - Is "_vectors dibuixats_" a good name for you?
  - You added `new costume %l width %dim height %dim` translation this January, but it had already been added on October. I've deleted your last one, but maybe you had done it intentionally? (my translation was (is) `'nou vestit %l d\'amplada %dim i alçada %dim'`but yours was without connectors.

Joan

### CODE REVIEW

**Comment by bromagosa** [comment]
I believe we both translated the `new costume` block in different PRs that got pulled. Your translation sounds better to me.

*Vectors dibuixats* sounds a bit funny, but I can't come up with a better translation right now :)

File: locale/lang-ca.js [hunk 1]
```diff
@@ -2723,7 +2723,23 @@ SnapTranslator.dict.ca = {
         'oposat',
     '__shout__go__':
         'bandera verda premuda',
-    'new costume %l width %dim height %dim':
-        'nou vestit %l amplada %dim alçada %dim'
+    'download script':
+        'descarrega el bloc de programa',
```
Comment (bromagosa) [inline locale/lang-ca.js]: Should be "descarrega el programa". This option downloads the whole script, not just a block.

**Comment by jguille2** [comment]
@jmoenig Now it's ok. Thanks,

Joan

**Comment by jmoenig** [comment]
Oh, I almost forgot about this PR, is this still open, i.e. should I pull it? Thanks!

**Comment by jguille2** [comment]
Yes! It is pending :)

**Comment by jmoenig** [comment]
thanks! I've marked it as "blessed", so I don't forget to pull it :-)

**Pull request merged**